website/docs/guides/best-practices/how-we-style/0-how-we-style-our-dbt-projects.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How we style our dbt projects
+id: 0-how-we-style-our-dbt-projects
+---
+
+## Why does style matter?
+
+Style might seem like a trivial, surface-level issue, but in fact it's a deeply material aspect of a well-built project. Consistent, clear style enhances readability, which in turn makes it easier to understand and maintain your project. Highly readable code will make it easier for you to build clear mental models of your project, which will make it easier for you to debug and extend your project. It's not just a favor to yourself though, equally importantly, it makes it easier for others to understand and contribute to your project, essential for peer collaboration, open source work, and onboarding new team members.
```

Comment:
Added a little variety to some of the adjectives to enhance the reading experience of the opening section. Let me know what you think
```suggestion
Style might seem like a trivial, surface-level issue, but it's a deeply material aspect of a well-built project. A consistent, clear style enhances readability and makes your project easier to understand and maintain. Highly readable code helps build clear mental models making it easier to debug and extend your project. It's not just a favor to yourself, though; equally importantly, it makes it effortless for others to understand and contribute to your project, which is essential for peer collaboration, open-source work, and onboarding new team members.
```